StitchCore/StitchCoreTests/TestHarness/TestHarness.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  TestHarness.swift
+//  StitchCoreTests
+//
+//  Created by Jason Flax on 2/1/18.
+//  Copyright © 2018 MongoDB. All rights reserved.
+//
+
+import Foundation
+@testable import StitchCore
+import PromiseKit
+import ExtendedJson
+
+internal let defaultServerUrl = "http://localhost:9090"
```

Comment:
We can do this later but this is going to want to default to prod or be lazily initialized based on environment settings. Same goes for username and password. For normal users, they can't even do username and password but only "api-key".

Reply:
👍